src/UNL/UCBCN/Calendar.php

```diff
@@ -97,6 +98,10 @@ function links()
                      'uidlastupdated' => 'users:uid');
     }
 
+    public function manageHref() {
+        return "/manager/" . $this->shortname . "/manage/";
```

Comment:
When the frontend and manager were two different code bases, I think it made sense to keep the URLs different.  Now that they are the same system, I think `/shortname/manage` isn't so bad.  Either way is fine.